sugoi-api-core/src/main/java/fr/insee/sugoi/core/service/GroupService.java

```diff
@@ -17,9 +17,9 @@
 import fr.insee.sugoi.core.exceptions.GroupNotCreatedException;
 import fr.insee.sugoi.core.exceptions.GroupNotFoundException;
 import fr.insee.sugoi.core.exceptions.UserNotFoundException;
-import fr.insee.sugoi.core.model.PageResult;
-import fr.insee.sugoi.core.model.PageableResult;
 import fr.insee.sugoi.model.Group;
+import fr.insee.sugoi.model.technics.PageResult;
```

Comment:
I don't like this `technics` package. If you really want to isolate those classe maybe `fr.insee.sugoi.model.paging` ?